core/app/services/spree/export/google/optional_attributes.rb

```diff
@@ -0,0 +1,21 @@
+module Spree
+  module Export
+    module Google
+      class OptionalAttributes
+        prepend Spree::ServiceModule::Base
+
+        def call(input)
+          information = {}
+
+          input[:settings].enabled_keys.each do |key|
```

Comment:
I'm not sure if we even need to store that in settings - I'm not sure if a regular Spree user (non-dev) won't find much value in being able to enable/disable a particular value.

From my perspective, we should be fine to implement it in a way that:
- by default, we try to take the value from product property
- if the value is not present in product properties, we don't return this value
- if someone wants to customize this behavior (either take it from another table, or avoid exposing a particular value), they can just provide a custom implementation of Spree::Export::Google::OptionalAttributes